public/app/core/components/sidemenu/SideMenu.tsx

```diff
@@ -24,7 +24,7 @@ export const SideMenu: FC = React.memo(() => {
   }
 
   return (
-    <nav className="sidemenu" data-testid="sidemenu">
+    <nav className="sidemenu" data-testid="sidemenu" aria-label="Navigation">
```

Comment:
I might be wrong but doesn't `nav` element have `navigation` role by default? I think in this case the aria-label is either redundant or should be more descriptive, e.g. `Side Menu Navigation`, in case there's more than one element with navigation role on a page.

Reply:
There is another nav element on the page, so I would probably go with `Main menu`. Skip any "navigation" wording, as the screen reader already reads it form the nav element.

Reply:
I will change the name :)